tests/pytest_tests/unit_tests/test_data_types.py

```diff
@@ -160,6 +160,7 @@ def test_captions(
     assert wandb.Image.all_captions([wbone, wbtwo]) == ["Cool", "Nice"]
 
 
+@pytest.mark.server_issues
```

Comment:
there is no mock server here... the run is mocked out if this fails, it means there is an issue with this test